lib/hydra/works/characterization/fits_document.rb

```diff
@@ -146,6 +146,14 @@ class FitsDocument
       t.frame_rate(proxy: [:metadata, :video, :frame_rate])
     end
 
+    # Cleanup phase; ugly name to avoid collisions
+    def __cleanup__
```

Comment:
What would you think of a couple of small changes to this method? Namely, something like:

```ruby
def __cleanup__
  # Check for conditions in a guard clause, in one fell swoop rather than two checks
  #   and don't bother modifying `file_mime_type` if it isn't in the problematic state (with a comma)
  return if file_mime_type.blank? || !file_mime_type.first.include?(',')
  # Not sure if `#send` is needed, but I'm A-OK with this method using it instead of the setter if so
  file_mime_type = [file_mime_type.first.split(','),first]
end
```